src/Gaffer/Loop.cpp

```diff
@@ -269,6 +269,12 @@ bool Loop::setupPlugs()
 	nextPlug()->setFlags( Plug::Dynamic, false );
 	previousPlug()->setFlags( Plug::Dynamic, false );
 
+	// Copy styling over from main plugs.
+	/// \todo It would be much better if colours could be registered once by
+	/// Plug TypeId, and then we wouldn't need to do this.
```

Comment:
+100000

Reply:
I've put this on my todo list for after this and the BoxOut refactoring is merged.